--- a/CMakeLists.txt	2017-04-06 12:38:13.000000000 +0200
+++ b/CMakeLists.txt	2017-04-28 15:09:09.000000000 +0200
@@ -123,9 +123,9 @@
 set(libdir ${prefix}/lib)
 set(includedir ${prefix}/include)
 get_filename_component(antlr3c_library_path "${ANTLR3C_LIBRARIES}" PATH)
-set(LIBS_PRIVATE "-L${antlr3c_library_path} -lantlr3c")
+set(LIBS_PRIVATE " -lantlr3c")
 get_filename_component(bctoolbox_library_path "${BCTOOLBOX_CORE_LIBRARIES}" PATH)
-set(LIBS_PRIVATE "${LIBS_PRIVATE} -L${bctoolbox_library_path} -lbctoolbox")
+set(LIBS_PRIVATE "${LIBS_PRIVATE} -lbctoolbox")
 if(ZLIB_FOUND)
 	set(REQUIRES_PRIVATE "${REQUIRES_PRIVATE} z")
 endif()

--- a/belle-sip.pc.in	2017-04-28 14:47:09.000000000 +0200
+++ b/belle-sip.pc.in 2017-04-28 15:08:05.000000000 +0200
@@ -9,5 +9,5 @@
 Requires.private: @REQUIRES_PRIVATE@ @TLS_PC@
 Version: @VERSION@
 Libs: -L@libdir@ -lbellesip
-Libs.private: @LIBS_PRIVATE@
+Libs.private: -L@CMAKE_INSTALL_FULL_LIBDIR@ @LIBS_PRIVATE@
 Cflags: -I@includedir@ 

